webapp/i18n/en.json

```diff
@@ -286,7 +284,7 @@
   "admin.email.selfPush": "Manually enter Push Notification Service location",
   "admin.email.skipServerCertificateVerification.description": "When true, Mattermost will not verify the email server certificate.",
   "admin.email.skipServerCertificateVerification.title": "Skip Server Certificate Verification: ",
-  "admin.email.smtpPasswordDescription": " Obtain this credential from administrator setting up your email server.",
+  "admin.email.smtpPasswordDescription": "The password for the SMTP server username.",
```

Comment:
@hmhealey @lfbrock I was asked previously by @jasonblais to make that text change. IT does read a bit odd though with the word "username" included. It could probably be dropped or maybe be changed to "associated with the username"